com.unity.shadergraph/Editor/Importers/ShaderGraphImporter.cs

```diff
@@ -22,7 +22,7 @@ namespace UnityEditor.ShaderGraph
     // This ifdef can be removed once V2 is the only option.
     [ScriptedImporter(101, Extension, 3)]
```

Comment:
Gotta bump both versions; Anything using Hybrid will have the top version number and those will need to re-import as well.